mscore/plugin/api/score.h

```diff
@@ -31,39 +31,54 @@ class Measure;
+      Q_PROPERTY(int                            harmonyCount      READ harmonyCount)
+      /** Whether score has harmonies (chord symbols) (read only).\n \since MuseScore 3.1 */
+      Q_PROPERTY(bool                           hasHarmonies      READ hasHarmonies)
+      /** Whether score has lyrics (read only).\n \since MuseScore 3.1 */
+      Q_PROPERTY(bool                           hasLyrics         READ hasLyrics)
+      /// Key signature at the start of the score, in number of accidentals,
+      /// negative for flats, postitive for sharps (read only).\n \since MuseScore 3.1
+      Q_PROPERTY(int                            keysig            READ keysig)
+      /** Last measure of the score (read only) */
+      Q_PROPERTY(Ms::PluginAPI::Measure*        lastMeasure       READ lastMeasure)
+      /** Last multimeasure rest measure of the score (read only).\n \since MuseScore 3.1 */
+      Q_PROPERTY(Ms::PluginAPI::Measure*        lastMeasureMM     READ lastMeasureMM)
+      /** Last score segment (read only) */
+      Q_PROPERTY(Ms::PluginAPI::Segment*        lastSegment       READ lastSegment) // TODO: make it function? Was property in 2.X, but firstSegment is a function...
+      /** Number of lyrics items (syllables) in the score (read only).\n \since MuseScore 3.1 */
+      Q_PROPERTY(int                            lyricCount        READ lyricCount)
+      /** Name of the score, without path leading to it and extension.\n \since MuseScore 3.1 */
+      Q_PROPERTY(QString                        name              READ name           WRITE setName)
+      /** Number of measures (read only) */
       Q_PROPERTY(int                            nmeasures         READ nmeasures)
-      /** Number of pages */
+      /** Number of pages (read only) */
       Q_PROPERTY(int                            npages            READ npages)
-      /** Number of staves */
+      /** Number of staves (read only) */
       Q_PROPERTY(int                            nstaves           READ nstaves)
-      /** Number of tracks */
+      /** Number of tracks (staves * 4) (read only) */
       Q_PROPERTY(int                            ntracks           READ ntracks)
-//       Q_PROPERTY(Ms::PageFormat*                pageFormat        READ pageFormat     WRITE undoChangePageFormat)
+//      Q_PROPERTY(Ms::PageFormat*                pageFormat        READ pageFormat     WRITE undoChangePageFormat)
       /** The list of parts */
       Q_PROPERTY(QQmlListProperty<Ms::PluginAPI::Part>     parts      READ parts)
-//       Q_PROPERTY(QString                        poet              READ poet)
-//       Q_PROPERTY(QString                        subtitle          READ subtitle)
-//       Q_PROPERTY(QString                        title             READ title)
+      /** Lyricist of score, as taken from the score properties.\n \since MuseScore 3.1 */
+      Q_PROPERTY(QString                        poet              READ poet)
```

Comment:
Maybe better name it `lyricist`?

Reply:
poet is basically for backwards compatibility, I believe a leftover from MuseScore 1